@dao-auction/hooks/useNounsProtocol.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react'
```

Comment:
@0xTranqui @Javier-Szyfer - so typechain is pretty cool for interacting with contracts - removes a lot of guesswork, and you can sculpt your contract calls in pretty custom ways as opposed to wagmi.